### ISSUE TITLE
utf8 flag missing

### ISSUE THREAD

**Issue by eserte** [DrHyde/perl-modules-Number-Phone#37]
At least the German area names are not properly encoded. Here's a sample (Donauwörth):
```
$ perl -MDevel::Peek -MNumber::Phone -e 'Dump(Number::Phone->new("+49 906 1234567")->areaname)'  
SV = PV(0x2565280) at 0x242ca10
  REFCNT = 1
  FLAGS = (TEMP,POK,pPOK)
  PV = 0x2121c40 "Donauw\303\266rth"\0
  CUR = 11
  LEN = 16
```
Correct output should be:
```
  ...
  FLAGS = (POK,pPOK,UTF8)
  PV = 0x11315c0 "Donauw\303\266rth"\0 [UTF8 "Donauw\x{f6}rth"]
  ...
```

**Comment by DrHyde** [comment]
Thanks. Does 5305eac0 fix it? It looks like it does to me, but I don't really understand all these Unicode shenanigans so shouldn't be trusted!

**Comment by eserte** [comment]
Yes, it works. Another possibility (probably slightly more efficient) would be to `use utf8;` and remove the backslashes before high bit characters, and no `map { ...Encode::decode... }`.

Another observation: with newest `ExtUtils::MakeMaker` value of the `PERL` variable in the generated `Makefile` is always quoted (even on Unix systems), so the last command in the build script fails:

```
./build-data.sh: line 110: "/usr/perl5.20.2p/bin/perl": No such file or directory
```

**Comment by DrHyde** [comment]
That's odd. Does `/usr/perl5.20.2p/bin/perl` exist on that machine? Because having extraneous quotes around an executable name shouldn't affect anything ...

```$ "/usr/bin/which" which
/usr/bin/which
$ /usr/bin/which which
/usr/bin/which
```

even in a script ...

```$ cat foo
#!/bin/sh
/usr/bin/which which
"/usr/bin/which" which
$ ./foo
/usr/bin/which
/usr/bin/which
```

**Comment by eserte** [comment]
But in backticks it's different:
```
$ `echo '/usr/bin/which' which`
/usr/bin/which
$ `echo '"/usr/bin/which"' which`
bash: "/usr/bin/which": No such file or directory
```
(Linux and FreeBSD, bash and zsh)

**Comment by DrHyde** [comment]
I fucking hate Unix sometimes!

**Comment by DrHyde** [comment]
Should be fixed in 3865fda5. Please test!

**Comment by eserte** [comment]
Tested 62b2ac54f9aa5479276f8799ec1b1058e281ff6e --- the Makefile re-build problem is fixed, and the changed implementation with `use utf8` looks good.
